index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link href="styles.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="hdiv">
+        <header class ="head">Uthman</header>
+        <i class="fa-solid fa-bars menu-bar"></i> 
+    </div> 
```

Comment:
- [ Required ] Please consider wrapping this section of your web page in a `header` tag. In most websites, the header usually contains the logo with a link to the homepage and a navigation menu, so the most ideal markup for this section would be as follows.
```HTML
<header class="hdiv">
  <a href="index.html" class ="head">Uthman</a>
  <nav><i class="fa-solid fa-bars menu-bar"></i> </nav>
</header> 
```
- [ ] The `a` tag serves as the application logo and a link to the homepage
- [ ] The `nav` tag serves as the navigation menu for the application and is also beneficial to persons using web assistive technologies such as screen readers to identify where the menu is.